reassembly/tcpassembly.go

```diff
@@ -703,6 +703,12 @@ func (a *Assembler) AssembleWithContext(netFlow gopacket.Flow, t *layers.TCP, ac
 			}
 			half.nextSeq = seq
 			action.queue = false
+		} else if t.PSH {
+			if *debugLog {
+				log.Printf("%v PSH encountered", key)
+			}
+			half.nextSeq = seq
```

Comment:
I don't htink we want to set nextSeq with this?

Reply:
I think nextSeq does need to get set.  The code I added is within the "if half.nextSeq == invalidSequence {" block a dozen lines up.  The PSH flag in this scenario is effectively another way to indicate a start if a SYN was missed (and *start was not overridden in Accept()).
I think this is not a common case, but I believe the code is correct.